.github/workflows/integration-tests-on-release.yaml

```diff
@@ -14,7 +14,6 @@ defaults:
 
 jobs:
   TestCephFlexSuite:
-    if: github.ref == 'refs/heads/master' || github.ref == 'refs/tags/release-*'
```

Comment:
Doesn't this just need to be 
`if: github.ref == 'refs/heads/master' || github.ref == 'refs/heads/release-*'`?

Reply:
Since at the top of the file the overall actions are only triggered on `push` to master or release-* branches, or if a tag is pushed, I don't see why we need this check here anyway.

Reply:
> Doesn't this just need to be
> `if: github.ref == 'refs/heads/master' || github.ref == 'refs/heads/release-*'` (i.e., **tags** changes to **heads**)?

right @BlaineEXE , the condition is wrong but we can remove this as @travisn said.